[Explanation] Arithmetic Progression (#107)
TheAlgorithms/Algorithms-Explanation: en/Basic Math/Arithmetic and Geometric Progressions.md

> # Arithmetic Progression

Reviewer: ```suggestion
# Arithmetic Progression

```

A sequence of numbers is said to be in an `Arithmetic progression` if the difference between any two consecutive terms is always the same. In simple terms, it means that the next number in the series is calculated by adding a fixed number to the previous number in the series. 

For example, 2, 4, 6, 8, 10 is an AP because the difference between any two consecutive terms in the series (common difference) is same (4 - 2 = 6 - 4 = 8 - 6 = 10 - 8 = 2).

<p align="center">
    <img width="60%" src="https://user-images.githubusercontent.com/75872316/122635132-ce38d100-d0ff-11eb-8fdf-2e14a9f640cc.png">
</p>

> **Facts about Arithmetic Progression :**

Reviewer: ```suggestion
**Facts about Arithmetic Progression:**
```

> 1. Initial term: In an arithmetic progression, the first number in the series is called the initial term.

Reviewer: ```suggestion
1. Initial term: In an arithmetic progression, the first number in the series is called the initial term.
```

2. Common difference: The value by which consecutive terms increase or decrease is called the `common difference`.

> 3. The behavior of the arithmetic progression depends on the common difference `d`. If the common difference is positive, then the members (terms) will grow towards positive infinity, but if the common difference is negative, then the members (terms) will grow towards negative infinity.

Reviewer: ```suggestion
3. The behavior of the arithmetic progression depends on the common difference `d`. If the common difference is positive, then the members (terms) will grow towards positive infinity. But if the common difference is negative, then the members (terms) will grow towards negative infinity.
```

> **Formula of nth term of an A.P :**

Reviewer: ```suggestion
**Formula of the nth term of an A.P:**
```

> If `a` is the initial term and `d` is the common difference.Thus, the explicit formula is:

Reviewer: ```suggestion
`a` is the initial term, and `d` is a common difference. Thus, the explicit formula is:
```

<p align="center">
    <img width="60%" src="https://user-images.githubusercontent.com/75872316/122635193-25d73c80-d100-11eb-9015-344d36633704.png">
</p>

> **Formula of sum of first n term of A.P:**

Reviewer: ```suggestion
**Formula of the sum of first nth term of A.P:**
```

<p align="center">
    <img width="60%" src="https://user-images.githubusercontent.com/75872316/122635260-7a7ab780-d100-11eb-82a5-8ceeba3aff03.png">
</p>

> **General Formulas to solve problems related to Arithmetic Progressions:** If `a` is the first term and `d` is the common difference:

Reviewer: ```suggestion
**General Formulas to solve problems related to Arithmetic Progressions:** If `a` is the first term and `d` too, that would be a common difference:
```
